CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 Change Log
 ==========
 
+## Version 0.24.0
+
+_2020-03-09_
+
+### Kotlin
+
+ * Introduce a view binding function that works with AndroidX ViewBindings. #985
+ * Rename ViewBinding to ViewFactory. #1009
+ * Rename ContainerHints to ViewEnvironment to match Swift. #1005
+
+### Swift
+
+ * Removes ViewRegistry, Introduce ViewControllerDescription. #974
+ * Introduce ViewEnvironment. #999
```

Comment:
Nit: For consistency with the other entries in this file, wrap the issue numbers in parentheses.

```suggestion
 * Introduce a view binding function that works with AndroidX ViewBindings. (#985)
 * Rename ViewBinding to ViewFactory. (#1009)
 * Rename ContainerHints to ViewEnvironment to match Swift. (#1005)

### Swift

 * Removes ViewRegistry, Introduce ViewControllerDescription. (#974)
 * Introduce ViewEnvironment. (#999)
```